package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "cancel-stuck-transactions",
```

Should we use a scoped package name? 

```suggestion
  "name": "@filecoin-station/cancel-stuck-transactions",
```

---

I had it scoped before but realized it has nothing to do with Station. If anything, it should be filecoin scoped

---

> OTOH, in the past, we liked the convention that npm package name is the same as the github org/repo slug, that's why I suggested this change.

I like the name to fit the repo name but I personally don't care which org it's in.

> Maybe https://github.com/space-meridian would be a better home & npm scope for this package?

You're right, it also doesn't fit in this org really. I think it would be nice to keep it next to the `spark-*` repos, because that's where it's needed the most.